app/views/layout_unbranded.html

```diff
@@ -1,24 +1,17 @@
-{#- We can't mount GOV.UK Frontend's assets at root as it's done automatically by the extensions framework. -#}
-{%- set assetPath = '/govuk/assets' -%}
-
-{% extends "govuk/template.njk" %}
+{% extends "layout.html" %}
 
 {% block headIcons %}
   <link rel="shortcut icon" href="{{ asset_path }}images/unbranded.ico?0.18.3" type="image/x-icon" />
-  <link rel="mask-icon" href="{{ asset_path }}images/gov.uk_logotype_crown.svg?0.18.3" color="#0b0c0c">
```

Comment:
Got rid of most of these icons since they're not needed and don't work anyway

Reply:
The icons are added when using `layout.html` as they are used by GOV.UK Design System template. Could we use `{{ super() }}` to add them here too?